e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -331,9 +331,9 @@ test('Example Imagery in Display layout', async ({ page }) => {
 
         return newImageCount;
     }, {
-        message: "verify that new images still stream in",
+        message: "verify that old images are discarded",
```

Comment:
looking at this, what is the expected behavior when the time control button `isPaused`? should a selected image that a user is looking at be discarded? or should we pause both incoming and removed images coming from the telemetry collection whilst paused? presumably when unpausing, we'd then resync with the telemetry collection (replace what we have with whatever is in the collection).

Reply:
@scottbell there's been some pretty big confusion as to what the expected behavior vs previous behavior vs current behavior vs DESIRED behavior has been with imagery. We should clear this up as a part of your PR so that the whole team is on the same page

Reply:
Talked with @charlesh88 and what we want paused to do is to not flip to the latest image. Pause does not mean images are not removed/added.